### PR TITLE
AspNetCore HealthCheckPublisher Added with IServiceCollection extension

### PR DESCRIPTION
**Pull request by fahrigoktuna**
Hi Sander,

I was thinking that Gauge metrics can be used for Microsoft.Extensions.Diagnostics.HealthChecks as an extensions. For that purpose, I'm creating pull request for AspNetCore HealthChecks inludes with README.md file changes.

### CODE REVIEW

File: Prometheus.AspNetCore/HealthChecks/HealthCheckPublisher.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            await Task.Run(() =>
+            {
+                foreach (var reportEntry in report.Entries)
+                    _healthCheckGaugeMetric.Labels(reportEntry.Key).
+                        Set((double)reportEntry.Value.Status);
+            });
+
+        }
+        public async Task PublishAsync(HealthReport report, CancellationToken cancellationToken) => await SendHealthReportMetrics(report);
+
```
Comment (sungam3r) [inline Prometheus.AspNetCore/HealthChecks/HealthCheckPublisher.cs:33]:
```suggestion
```

File: Prometheus.AspNetCore/Prometheus.AspNetCore.csproj [hunk 2]
```diff
@@ -40,6 +40,11 @@
     <PackageReference Include="Microsoft.AspNetCore" Version="2.1.7" />
   </ItemGroup>
   
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="3.1.3" />
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions" Version="3.1.3" />
```
Comment (sungam3r) [inline Prometheus.AspNetCore/Prometheus.AspNetCore.csproj:45]:
```suggestion
```

**Comment by sandersaares** [comment]
This looks good. I request two enhancements:

1. To ensure that people who do not use HealthChecks do not need to take a new dependency, it would be better to separate this into a prometheus-net.AspNetCore.HealthChecks NuGet package.
2. It would be useful to have at minimum a sample app that can be easily used by a developer to try out this functionality and execute it without having to set up a separate solution (along the lines of Tester.NetFramework.AspNet).

Do you have the availability to perform these enhancements?

**Comment by sandersaares** [comment]
OK, I'll take care of it myself.

**Comment by sandersaares** [comment]
Thank you for the contribution! I have made some small adjustments and merged this.

**Comment by sandersaares** [comment]
This is now available in preview NuGet package version 3.6.0-pre-000129-bd91778. Please give it a try. If no problems are reported within a few days, I will publish this build as stable package.

**Pull request merged**